### ISSUE TITLE
Config to get scholarly to return >20 coauthors in filled Author (running in Docker with chromedriver)

### ISSUE THREAD

**Issue by mzhukovs** [scholarly-python-package/scholarly#411]
I am using scholarly inside a custom container for Azure Functions (running on Linux) and would like it to pull more than the standard 20 co-authors. I have installed the dependencies and added to PATH, and tested to ensure that selenium can in fact run now. 

However, I am still only getting 20 co-authors back. Is there another config or parameter I am missing to enable this?

Relevant snippet of Dockerfile:
```

ARG CHROME_VERSION="google-chrome-stable"
RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
    && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
    && apt-get update -qqy \
    && apt-get -qqy install \
    ${CHROME_VERSION:-google-chrome-stable} \
    && rm /etc/apt/sources.list.d/google-chrome.list \
    && rm -rf /var/lib/apt/lists/* /var/cache/apt/*

RUN LATEST=$(wget -q -O - http://chromedriver.storage.googleapis.com/LATEST_RELEASE) && \
    wget http://chromedriver.storage.googleapis.com/$LATEST/chromedriver_linux64.zip && \
    unzip chromedriver_linux64.zip && ln -s $PWD/chromedriver /usr/local/bin/chromedriver

ENV PATH="/usr/local/bin/chromedriver:${PATH}"
```

I have also upgraded to v1.6.0 and still only 20 co-authors get returned in a filled Author object, e.g. for author ID cCXnE7IAAAAJ - please let me know when you get a chance. Thank you very much.

**Comment by ipeirotis** [comment]
Could you please give the code that you use?

**Comment by mzhukovs** [comment]
Sure thing, just basic use:

```python

        sections_to_include = ["basics", "indices", "counts", "coauthors"]

        author: Author = scholarly.fill(
            scholarly.search_author_id(authorId),
            sections=sections_to_include,
            publication_limit=pub_limit,
        )

        return author
```

**Comment by ipeirotis** [comment]
@arunkannawadi Do you think that we need the selenium/geckodriver moving forward?

I noticed that if we go to `https://scholar.google.com/citations?view_op=list_colleagues&hl=en&user=cCXnE7IAAAAJ` we get the full list of co-authors, without having to use selenium.

**Comment by arunkannawadi** [comment]
Ooh, how did you stumble upon that link? Yes, fetching the complete co-author list is the only place where we use selenium, and this link makes it redundant. It should work much nicer in containerized environments as well and fetch the lost much faster.

**Comment by mzhukovs** [comment]
Wow - indeed a great find.... that would be ideal - if there's a solution in the meantime to work with Selenium please let me know, otherwise really looking forward to this update

**Comment by arunkannawadi** [comment]
You're welcome to install scholarly with `install https://github.com/scholarly-python-package/scholarly.git@develop` and give it a try. You might be able to fetch the complete list of coauthors. We might want to do some more testing before tagging another release.

**Comment by arunkannawadi** [comment]
Sorry, I commented too soon. Even the `develop` branch right now won't solve you issue, but we will have another version in a week or so that should allow you to fetch all coauthors.

**Comment by mzhukovs** [comment]
That's really great - awesome work really appreciate your efforts all. Thank you very much

**Comment by mzhukovs** [comment]
@arunkannawadi awesome to see the release already - ETA on being available on Pypy? Still see only 1.6.0 there https://pypi.org/project/scholarly/

**Comment by arunkannawadi** [comment]
I'm not sure why PyPI is not picking up the new version. I've pushed it twice by now and our conda option is also linked to PyPI. You can install it directly from Github using `pip install -U` (see README.md) and you'll get the recent version.

**Comment by ipeirotis** [comment]
We should be fine now. There was a change of the script that was publishing to PyPI last month, which did not work as intended. Reverted it back and released 1.6.2

**Comment by mzhukovs** [comment]
Thanks again guys!!
